tests/subsys/logging/log_stack/src/test.inc

```diff
@@ -0,0 +1,288 @@
+/*
```

Comment:
Suffix should be .h, not .inc.

This makes it easier for tools and humans to understand what kind of file it is.

Reply:
there is a pattern, first started in `tests/unit/util` to name if `test.inc` and this is forth test that do that. I would keep it as is for consistency.

Reply:
fair enough